pkg/controllers/node/health/suite_test.go

```diff
@@ -186,6 +187,23 @@ var _ = Describe("Node Health", func() {
 			nodeClaim = ExpectExists(ctx, env.Client, nodeClaim)
 			Expect(nodeClaim.Annotations).To(HaveKeyWithValue(v1.NodeClaimTerminationTimestampAnnotationKey, fakeClock.Now().Format(time.RFC3339)))
 		})
+		It("should not update termination grace period if set past the current time", func() {
```

Comment:
nit on the wording, seems like you're saying "past" indicating how far in the past it is
```suggestion
		It("should not update termination grace period if set before the current time", func() {
```